src/core/lib/iomgr/ev_poll_posix.cc

```diff
@@ -233,9 +233,8 @@ static int poll_deadline_to_millis_timeout(grpc_core::Timestamp deadline);
 #define GRPC_POLLSET_REEVALUATE_POLLING_ON_WAKEUP 2
 // As per pollset_kick, with an extended set of flags (defined above)
 // -- mostly for fd_posix's use.
-static grpc_error_handle pollset_kick_ext(grpc_pollset* p,
-                                          grpc_pollset_worker* specific_worker,
-                                          uint32_t flags) GRPC_MUST_USE_RESULT;
+GRPC_MUST_USE_RESULT static grpc_error_handle pollset_kick_ext(
```

Comment:
I think GRPC_MUST_USE_RESULT is unnecessary here, since `grpc_error_handle` is absl::Status

Reply:
fixed